tslint.json

```diff
@@ -4,6 +4,7 @@
         "interface-name": [true, "never-prefix"],
         "max-classes-per-file": false,
         "member-ordering": false,
+        "no-console": [true, "log"],
```

Comment:
So there is a lint which checks if you use console? Why is it considered bad?

Reply:
See https://palantir.github.io/tslint/rules/no-console/. I agree for web code, but I think for nodejs it's the way to go.